config/locales/exceptions.en.yml

```diff
@@ -128,6 +128,12 @@ en:
         detail: "Did not receive a timely response from an upstream server"
         code: 504
         status: 504
+      sentry_ignored_gateway_timeout:
+        <<: *defaults
```

Comment:
Good catch.  How about using inheritance here so the two gateway_timeout errors don't diverge?
Declare `gateway_timeout: &gateway_timeout` on line 125, then here just:
`<<: *gateway_timeout`